modules/advangelistsBidAdapter.js

```diff
@@ -0,0 +1,443 @@
+import * as utils from '../src/utils';
+import { parse as parseUrl } from '../src/url';
+import { config } from '../src/config';
+import { registerBidder } from '../src/adapters/bidderFactory';
+// import { Renderer } from '../src/Renderer';
+import { VIDEO, BANNER } from '../src/mediaTypes';
+import find from 'core-js/library/fn/array/find';
+import includes from 'core-js/library/fn/array/includes';
+
+const ADAPTER_VERSION = '1.0';
+const BIDDER_CODE = 'avng';
+// const OUTSTREAM = 'outstream';
+
+export const VIDEO_ENDPOINT = '//nep.advangelists.com/xp/get?pubid=';// 0cf8d6d643e13d86a5b6374148a4afac';
+export const BANNER_ENDPOINT = '//nep.advangelists.com/xp/get?pubid=';// 0cf8d6d643e13d86a5b6374148a4afac';
+export const OUTSTREAM_SRC = '//player-cdn.beachfrontmedia.com/playerapi/loader/outstream.js';
+
+export const VIDEO_TARGETING = ['mimes', 'playbackmethod', 'maxduration', 'skip'];
+export const DEFAULT_MIMES = ['video/mp4', 'application/javascript'];
+
+let pubid = '';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER, VIDEO],
+
+  isBidRequestValid(bidRequest) {
+    if (typeof bidRequest != 'undefined') {
+      if (bidRequest.bidder !== BIDDER_CODE && typeof bidRequest.params === 'undefined') { return false; }
+      if (bidRequest === '' || bidRequest.params.placement === '' || bidRequest.params.pubid === '') { return false; }
+      return true;
+    } else { return false; }
+  },
+
+  buildRequests(bids, bidderRequest) {
+    let requests = [];
+    let videoBids = bids.filter(bid => isVideoBidValid(bid));
+    let bannerBids = bids.filter(bid => isBannerBidValid(bid));
+    videoBids.forEach(bid => {
+      pubid = getVideoBidParam(bid, 'pubid');
+      requests.push({
+        method: 'POST',
+        url: VIDEO_ENDPOINT + pubid,
+        data: createVideoRequestData(bid, bidderRequest),
+        bidRequest: bid
+      });
+    });
+
+    bannerBids.forEach(bid => {
+      pubid = getBannerBidParam(bid, 'pubid');
+      requests.push({
+        method: 'POST',
+        url: BANNER_ENDPOINT + pubid,
+        data: createBannerRequestData(bid, bidderRequest),
+        bidRequest: bid
+      });
+    });
+
+    console.log(requests);
+    return requests;
+  },
+
+  interpretResponse(serverResponse, {bidRequest}) {
+    let response = serverResponse.body;
+    if (response !== null && utils.isEmpty(response) == false) {
+      if (isVideoBid(bidRequest)) {
+        console.log('About to create final response for video')
```

Comment:
Remove `console.log` call, otherwise it will pollute console when running tests/etc

Reply:
Thanks. Removed console.log.